src/widgets/keyboard/lv_keyboard_pinyin.c

```diff
@@ -0,0 +1,999 @@
+
+/**
+ * @file lv_keyboard_pinyin.c
```

Comment:
Would `lv_pinyin_default_dict.c/.h` be better?

Reply:
> Would `lv_pinyin_default_dict.c/.h` be better?

Whether the file name needs to be related to the folder name, such as prefixed with `lv_keyboard_`, But `lv_keyboard_pinyin_default_dict.c/.h` is too long